README.md

```diff
@@ -64,6 +64,20 @@ Flavors can be built in parallel easily with the `build.parallel` target:
 make FLAVORS="<flavor> <other flavor> <...>" build.parallel
 ```
 
+### Building with a specific version of Ceph
```

Comment:
Please follow exising readme style, split lines to 100 chars or less, and use single grave-accents.